components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -151,6 +153,42 @@ public static void writeToPropertyFile(Properties properties, String filePath) {
         }
     }
 
+    /**
+     * Retrieve the value for the given configuration from the following order of files.
+     * 1. deployment.toml
+     * 2. default.json
+     * 3. carbon.xml
+     *
+     * @param config        configuration value

```

Comment:
Attribute description should start with a capital letter and stop with a full stop> Look for other places as well.
`@param config        Configuration value.`

Reply:
Addressed by https://github.com/wso2/cipher-tool/pull/76